streams/src/main/java/org/apache/kafka/streams/processor/TopologyBuilder.java

```diff
@@ -243,7 +247,36 @@ public final TopologyBuilder addSink(String name, String topic, String... parent
 
     /**
      * Add a new sink that forwards messages from upstream parent processor and/or source nodes to the named Kafka topic.
+     * The sink will use the {@link StreamingConfig#KEY_SERIALIZER_CLASS_CONFIG default key serializer} and
+     * {@link StreamingConfig#VALUE_SERIALIZER_CLASS_CONFIG default value serializer} specified in the
+     * {@link StreamingConfig streaming configuration}.
+     * <p>
+     * The sink will also use the specified {@link StreamPartitioner} to determine how messages are distributed among
+     * the named Kafka topic's partitions. Such control is often useful with topologies that use
+     * {@link #addStateStore(StateStoreSupplier, String...) state stores}
+     * in its processors. In most other cases, however, a partitioner need not be specified and Kafka will automatically distribute
+     * messages among partitions using Kafka's default partitioning logic.
```

Comment:
Can you be more specific? There is already a method matching that signature on [line 244](https://github.com/apache/kafka/pull/309#discussion-diff-49125479R244), just before this method?